runtime/libffi/z/ffi64.c

```diff
@@ -919,6 +921,16 @@ ffi_prep_cif_machdep(ffi_cif *cif)
       /* Check how a structure type is passed.  */
       if (type == FFI_TYPE_STRUCT)
 	{
+    /* ffi_chcek_struct_type will return FFI_TYPE_UINT64 for structure with
+        exactly two floating point values of short format which is still OK as
+        in calculation for how many bytes needed in the stack, it will count GPR
+        for such parameters and one double word per argument will be requested.
```

Comment:
Please respect the formatting of this file:
* it appears to assume tabs are equivalent to eight spaces
* code is indented two spaces per level - one level for `{` and a second for code within a block

Please correct the spelling of `ffi_check_struct_type()` and use parentheses to clarify the reference is to a function.

This comment appears to be missing the occasional word:
* "for structure" should be "for a structure"
* "are and set" should be "are needed and set"

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/compare/c2018048ad37140e438aec106082576687394dd6..b6043edad9cc1c6681422ab75fac8bf576b3dde3 and https://github.com/eclipse-openj9/openj9/compare/5936d79b5911f033286a0fabca8f4f56278e4d29..2bd00bfe772af4a4d85446a336a0ddd079b94090